src/tablo/tabloVariables.scss

```diff
@@ -3,12 +3,38 @@
 $border-color: $brc-silver !default;
 $border-width: 1px !default;
 
+$table-border-color: $border-color !default;
+$table-border-width: $border-width !default;
+$table-border-left-color: $table-border-color !default;
```

Comment:
There are cases where you don't want the bottom border or the right border.
Of course it's hard to find a case when one would use all those variables, but with the default it should work well with minimal setup